src/main/java/com/itech/kinopoisk/exceptions/TooLowAccessException.java

```diff
@@ -0,0 +1,19 @@
+package com.itech.kinopoisk.exceptions;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@NoArgsConstructor
+@Getter
+@ResponseStatus(code = HttpStatus.NOT_FOUND)
+public class TooLowAccessException extends RuntimeException {
+
+    private String string;
```

Comment:
why variable name is "string"? Should be more meaningful. "message" or something else